joplin/templates/wagtailadmin/pages/listing/_table_headers_explore.html

```diff
@@ -20,16 +20,16 @@
         <th class="ord">
```

Comment:
I see what you're doing here and it makes sense. However, rather than edit this table_headers_explore file, I think we should edit/extend the template used by wagtail for search. We're now turning this template into something that it isn't, and it isn't going to be clear in the future why we did that and may cause future compatibility problems or make it difficult to switch back. 

At first glance it looks like search is using this template, perhaps? 
https://github.com/wagtail/wagtail/blob/23a89774adef4f609e33eb3feece90d6535588ee/wagtail/admin/templates/wagtailadmin/pages/search_results.html

Reply:
So that is the wagtail template file that I did pull in here: 
- ttps://github.com/cityofaustin/joplin/pull/418/files?file-filters%5B%5D=.html&file-filters%5B%5D=.py&hide-deleted-files=true#diff-8d43fac787c5b9a5e1af51737d716906

In addition the `_table_headers_explore.html` file was also pulled in from wagtails templates here: 
- https://github.com/wagtail/wagtail/blob/23a89774adef4f609e33eb3feece90d6535588ee/wagtail/admin/templates/wagtailadmin/pages/listing/_table_headers_explore.html
- Then, I only modified the url when sorting to *included* the search query so it didn't revert back to *all* pages when sorting.

Reply:
I'll add notes to that file being pulled in so it's clear where it came from.